cuda_core/cuda/core/experimental/_device.py

```diff
@@ -54,15 +81,24 @@ def _check_context_initialized(self, *args, **kwargs):
 
     @property
     def device_id(self) -> int:
+        """Return device ordinal."""
         return self._id
 
     @property
     def pci_bus_id(self) -> str:
+        """Return a PCI Bus Id string for this device."""
         bus_id = handle_return(cudart.cudaDeviceGetPCIBusId(13, self._id))
         return bus_id[:12].decode()
 
     @property
     def uuid(self) -> str:
+        """Return a UUID for the device.
+
+        Returns 16-octets identifying the device. If the device is in
+        MIG mode, returns its MIG UUID which uniquely identifies the
+        subscribed MIG compute instance.
```

Comment:
Very cool - did you check this works fine in the MIG mode?

Reply:
No but I'm taking that comment directly from the `cuDeviceGetUuid_v2` header documentation. Note that `cuDeviceGetUuid` doesn't do this, but we only use it for old CUDA versions (<11.4).

I was on the fence about adding it to docstring, but decided to do so in the latest commit.

Reply:
I like it included